_posts/2021-02-11-pif-18f-usds.md

```diff
@@ -0,0 +1,80 @@
+
+18F and PIF are based in the General Services Administration, a support agency for the executive branch (for example, it manages most federal real estate and manages large contract vehicles). 18F and PIF have hoteled office space in GSA's headquarters a block from the White House at 1800 F street (the [origin of 18F's name](https://18f.gsa.gov/about/#history-and-funding)).
+
+### How their approaches complement each other
+
+When you combine the differences above, each organization ends up with unique benefits and access by virtue of their different approaches and methodologies.
+
+- Thanks to its EOP ties, USDS has unique access to presidential priorities or sensitive Cabinet level programs, and being appropriated means it can spin up on projects immediately.
+- Thanks to its GSA ties, 18F can work with teams in the legistlative and judicial branches, as well as with independent regulatory agencies that keep at arm's length from the White House like the Federal Election Commission. Bluntly, GSA has a distance and boring-ness that allows 18F to avoid a perception of executive encroachment on other branches.
+- Thanks to their "in residence" nature, PIFs can flexibly jump to other projects within their host agencies in a more "free agent" style over the course of their terms.
+
+### Collaboration tools and tech
+
+USDS often works with agencies just getting started with cloud tooling such as Microsoft Teams, so a USDSer enjoys all of the same frustrations as their career civil servant partners when trying to collaborate on a document or access shared information. GSA uses GSuite across its enterprise (the GSA IT team made the agency a [first-mover to cloud email](https://www.gsa.gov/about-us/newsroom/news-releases/gsa-becomes-first-federal-agency-to-move-email-to-the-cloud-agencywide) in 2010), so 18F and PIF teams enjoy real-time collaboration on documents with anyone at GSA. Like USDS, PIF teams get equipment and e-mail addresses from the agencies where they are deployed - there can be significant friction if you try to work with one agency using a laptop from a different agency.
+
+### Engagement incubation
+
+USDS’s appropriation allows it to jump into supporting an agency on a problem immediately, though work often starts with a [discovery sprint](https://www.linkedin.com/pulse/government-discovery-sprint-playbook-how-lessons-learned-kathy-pham/). 18F sets up a formal agreement before working on an engagement, which requires signoff from multiple partner officials and in my experience, usually took over a month to execute. PIFs are deployed to agencies annually, and agencies submit projects for PIFs months in advance to be considered for that year’s class of fellows under a similar agreement to 18F's. It’s also common for PIFs to flex to other special projects once they start at an agency, based on needs that come up throughout the year.
```

Comment:
Worth adding a link to 18F's `How we work` in this section? https://18f.gsa.gov/how-we-work/

Reply:
great call!